iaso/api/devices_position.py

```diff
@@ -55,8 +55,8 @@ class DevicesPositionViewSet(ModelViewSet):
 
     This API is open to anonymous users on write (if the project allows it).
 
-    GET /api/devicesposition/
-    POST /api/devicesposition/
+    GET /api/devicespositions/
+    POST /api/devicespositions/
```

Comment:
The mobile application uses this one. It should still be callable without the `s` (at least for the `POST`)